db/migrate/20160523155343_add_enabled_column_to_service.rb

```diff
@@ -0,0 +1,5 @@
+class AddEnabledColumnToService < ActiveRecord::Migration
+  def change
+    add_column :services, :enabled, :boolean, default: false, null: false
```

Comment:
A good question.  Part of me thinks: "yeah put an index on it because we'll be filtering by this a lot", but part of me thinks "no need, it's a boolean field so there's only two values, surely an index won't help" - so I'm not sure.  Did some digging (on stackoverflow, obvs) and found this http://stackoverflow.com/questions/12539772/adding-an-index-on-a-boolean-field which suggests a partial index might be useful?  Never heard of partial indexes before so that's exciting.

In conclusion: maybe?